### PR TITLE
new package: slugify

### PR DESCRIPTION
**Pull request by Freed-Wu**

### CODE REVIEW

**Review by xtkoba — CHANGES_REQUESTED** [review]
We cannot legally package a software that does not specify any license. Please use a "git" version that contains the license file.

File: packages/slugify/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=f6873b062119d3eaa7d89254fc6e241debf074da02e3189f12e08b372af096e5
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS=bash
+
+termux_step_make_install() {
+	cd "$TERMUX_PKG_SRCDIR"
+	install -D slugify -t "$TERMUX_PREFIX/usr/bin"
+	install -D slugify.1 -t "$TERMUX_PREFIX/usr/share/man/man1"
```
Comment (xtkoba) [inline packages/slugify/build.sh:15]: These directories should be `$TERMUX_PREFIX/bin` and `$TERMUX_PREFIX/share/man/man1` respectively.

**Comment by Freed-Wu** [comment]
OK. LICENSE had been added.

**Comment by Freed-Wu** [comment]
Fix bug of /usr

File: packages/slugify/build.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+TERMUX_PKG_DEPENDS=bash
+
+termux_step_make_install() {
+	cd "$TERMUX_PKG_SRCDIR"
+	install -D slugify -t "$TERMUX_PREFIX/bin"
+	install -D slugify.1 -t "$TERMUX_PREFIX/share/man/man1"
+}
+
+termux_step_install_license() {
+	curl https://github.com/benlinton/slugify/raw/master/LICENSE > LICENSE
```
Comment (xtkoba) [inline packages/slugify/build.sh:19]: This is not acceptable. If `LICENSE` is added after the release then it is not applied to that version.

Moreover, if you download external files, you should use `termux_download` and calculate the checksum.

**Review by xtkoba — CHANGES_REQUESTED** [review]

**Comment by Freed-Wu** [comment]
I see. How to use git to download the latest source?

**Comment by xtkoba** [comment]
There are several ways, but I adopt the one shown in for example https://github.com/termux/termux-packages/blob/d8e0b32c11923b82ec5072f18c4c162a12ce1822/packages/2048-c/build.sh.

**Comment by Freed-Wu** [comment]
OK

File: packages/slugify/build.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/benlinton/slugify
+TERMUX_PKG_DESCRIPTION="Bash command that converts filenames and directories to a web friendly format."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE.git
+TERMUX_PKG_VERSION=1.0.1
+TERMUX_PKG_GIT_BRANCH=master
```
Comment (xtkoba) [inline packages/slugify/build.sh:7]: These two lines are inconsistent with each other. `master` is not version `1.0.1`.

**Review by xtkoba — CHANGES_REQUESTED** [review]

**Comment by xtkoba** [comment]
Alternatively you can wait for the next (upstream) version to be released.

**Comment by Freed-Wu** [comment]
Thanks. I have known how to use git to download the latest source.

**Comment by Freed-Wu** [comment]
Thanks. `$_COMMIT` has been added.

**Review by xtkoba — APPROVED** [review]

**Pull request merged**